metaflow/plugins/aws/batch/batch_cli.py

```diff
@@ -47,10 +47,10 @@ def _execute_cmd(func, flow_name, run_id, user, my_runs, echo):
     if my_runs:
         user = util.get_username()
 
-    latest_run = False
+    latest_run = True
```

Comment:
What is this change?

Reply:
The previous behaviour didn't work as expected.